### PR TITLE
Get proper log-api url for monasca-log-agent

### PR DESCRIPTION
**Pull request by matrixik**

### CODE REVIEW

File: chef/cookbooks/monasca/libraries/helper.rb [hunk 1]
```diff
@@ -25,13 +25,20 @@ def self.monasca_admin_host(node)
...
+  def self.api_public_url(node)
+    host = monasca_public_host(node)
     protocol = node[:monasca][:api][:ssl] ? "https" : "http"
     port = node[:monasca][:api][:bind_port]
     "#{protocol}://#{host}:#{port}/v2.0"
   end
 
+  def self.log_api_public_url(node)
+    host = monasca_public_host(node)
+    protocol = node[:monasca][:api][:ssl] ? "https" : "http"
```
Comment (artur-ba) [inline chef/cookbooks/monasca/libraries/helper.rb]: shouldn't be : node[:monasca][:log_api][:ssl] ??

Comment (matrixik) [reply chef/cookbooks/monasca/libraries/helper.rb]: done

**Review by samirjorina — APPROVED** [review]

**Review by kornicameister — COMMENTED** [review]
Shouldn't you remove log api URL from log_agent settings now ?

File: chef/cookbooks/monasca/libraries/helper.rb [hunk 2]
```diff
@@ -32,6 +32,13 @@ def self.api_public_url(node)
     "#{protocol}://#{host}:#{port}/v2.0"
   end
 
+  def self.log_api_public_url(node)
+    host = monasca_public_host(node)
+    protocol = node[:monasca][:log_api][:ssl] ? "https" : "http"
```
Comment (kornicameister) [inline chef/cookbooks/monasca/libraries/helper.rb]: I think we don't have **SSL** support at this point.
There are 2 things, at least, that we should do to have it:

* pass that information to ansible role
* make ansible role capable of configuring ssl

Are we sure we have that ?

Comment (matrixik) [reply chef/cookbooks/monasca/libraries/helper.rb]: Now I looked into it and no, we don't have that, removing.

Comment (matrixik) [reply chef/cookbooks/monasca/libraries/helper.rb]: But I will leave configuration option in data_bag for future.

**Review by kornicameister — APPROVED** [review]
Go head...squash and be happy.

**Pull request merged**